src/components/App/App.js

```diff
@@ -62,7 +62,25 @@ export default class App extends Component {
           logOut={this.logOut}
           movies={this.state.movies}
           addMovies={this.addMovies}/>} />
-          
+
+        <Route path='/movies/:id' render={({ match }) => {
+          console.log("MATCH", match)
+          const specificMovie = this.state.movies.find(info => info.episode_id === parseInt(match.params.id))
+          console.log("MOOOVIE", this.state.movies)
```

Comment:
Can you remove all console.logs before merge?

Reply:
gaahhh whoops